ban/http/api.py

```diff
@@ -391,18 +408,31 @@ def get_version(self, identifier, ref):
     @app.jsonify
     @app.endpoint('/<identifier>/versions/<int:ref>/flag', methods=['POST'])
     def post_version(self, identifier, ref):
-        """Flag a version.
+        """Flag a {resource} version.
 
         parameters:
             - $ref: '#/parameters/identifier'
             - name: ref
               in: path
-              type: string
+              type: integer
               required: true
               description: version reference, either a date or an increment.
+            - name: body
+              in: body
+              schema:
```

Comment:
I think the schema is wrong here: it's pointing to the resource, while it's just a `{status: xxx}` one.